src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/DevAdventCalendarCompetition.Tests.csproj

```diff
@@ -2,10 +2,21 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
+    <CodeAnalysisRuleSet>..\Analysers.ruleset</CodeAnalysisRuleSet>
```

Comment:
please use AnalysersTests.ruleset here (if it does not exist, create it as a copy of Analysers.ruleset)